examples/knative/powershell/kn-ps-mm-sync/kn-ps-mm-sync.yml

```diff
@@ -0,0 +1,57 @@
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: kn-ps-mm-sync-service
+  labels:
+    app: veba-ui
+  namespace: vmware-functions
+spec:
+  template:
+    metadata:
+      annotations:
+        autoscaling.knative.dev/maxScale: "1"
+        autoscaling.knative.dev/minScale: "1"
+    spec:
+      containers:
+        - image: ghcr.io/darrylcauldwell/kn-ps-mm-sync-image:1.0
```

Comment:
Replace with `projects.registry.vmware.com/veba/kn-ps-mm-sync:1.0` and once approved, we'll build this image and push to VEBA repository